### PR TITLE
Ck 43 chore creation page

### PR DESCRIPTION
**Pull request by Amikz**
Made the chore creation page and allow for the creation of chores. You can complete the information and save the chore.
image

### CODE REVIEW

**Review by Tosvng — APPROVED** [review]
This looks great!

**Review by JoshuaDueck — APPROVED** [review]
Looks good! We'll likely need to iron out some styling and imports once things are put together, but I'm happy we've got a chore creation form. Thanks for your work!

File: frontend/src/components/CreateChore.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import { React, useState } from "react";
+import { Button } from "primereact/button";
+import Modal from 'react-bootstrap/Modal';
```
[the remark this answers is not in the document]
Comment (Amikz) [reply frontend/src/components/CreateChore.js:3]: I saw that it was a thing when I went to add the prime react stuff, but I liked the way the bootstrap one looked better

[the remark this answers is not in the document]
Comment (Amikz) [reply frontend/src/components/CreateChore.js:3]: Okay, I can look into it some more later if we have time

**Pull request merged**